apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/RowAsyncEnumerator.cs

```diff
@@ -321,18 +321,20 @@ public override RowMergeState HandleChunk(RowAsyncEnumerator owner, CellChunk ch
                 }
                 owner.Assert(chunk.ValueSize >= 0, "NewCell valueSize can't be negative");
 
+                bool familyNameChanged = false;
                 if (chunk.FamilyName != null)
                 {
                     if (chunk.FamilyName != owner._currentCell.Family?.Name)
                     {
                         owner._currentCell.Family = new Family { Name = chunk.FamilyName };
                         Debug.Assert(!owner._currentFamilies.ContainsKey(chunk.FamilyName));
                         owner._currentFamilies[chunk.FamilyName] = owner._currentCell.Family;
+                        familyNameChanged = true;
                     }
                     owner.Assert(chunk.Qualifier != null, "NewCell has a familyName, but no qualifier");
                 }
 
-                if (chunk.Qualifier != null && chunk.Qualifier != owner._currentCell.Column?.Qualifier)
+                if (chunk.Qualifier != null && (chunk.Qualifier != owner._currentCell.Column?.Qualifier || familyNameChanged))
```

Comment:
What should happen if chunk.Qualifier is null, but the family name has changed? (I don't know enough about the underlying mechanics to know if that's something we should handle.)

Reply:
My understanding is that this if condition determines if the current cell should be associated with a new column or not. If the family has changed and qualifier is null, it should be an invalid scenario as per this test case that states that a new column family must have a qualifier.  https://github.com/Rishabh-V/conformance-tests/blob/main/bigtable/v2/readrows.json#L50

Reply:
Works for me :)

Reply:
Thanks Jon. I will wait for Igor to have a look as well. Meanwhile, [conformance tests](https://github.com/googleapis/conformance-tests/pull/74) with new test cases are failing, so that may require some more time to be merged.